jambi/config.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from jambi.exceptions import ImproperlyConfigured
+
+ENVIRONMENT_VARIABLE = "JAMBI_CONFIG"
```

Comment:
Absolute.  It just gets passed to the config parser as-is, assuming we can verify it's a file.